travis/build_info.py

```diff
@@ -104,23 +122,67 @@ def repo_active_build_count(builds):
     return build_count, started_count
 
 
-def repo_active_job_count(active_builds):
+def get_job_counts(org):
+    """
+    Total job counts (active and total) for
+    an org
+    """
+    total_job_count = 0
+    total_started_job_count = 0
+
+    repos = get_repos(org)
+    for repo in repos:
+        repo_builds = get_active_builds(org, repo)
+        repo_jobs = 0
+        repo_started_jobs = 0
+        for build in repo_builds:
+            build_jobs = get_active_jobs(build['id'])
+            total, started = active_job_counts(build_jobs)
+            total_job_count += total
+            total_started_job_count += started
+            repo_jobs += total
+            repo_started_jobs += started
+        logger.debug("----> " + repo)
+        debug_msg = "total jobs: {total}, started jobs: {started}".format(
+            total=repo_jobs,
+            started=repo_started_jobs
+        )
+        logger.debug(debug_msg)
+    logger.debug('--------')
+    logger.info('overall_jobs_total=' + str(total_job_count))
+    logger.info('overall_jobs_started=' + str(total_started_job_count))
+
+
+def get_build_counts(org):
     """
-    Returns count of active jobs associated with builds in a
-    given repo.
+    Find out all the active and waiting builds for a given
+    Github/Travis org
     """
-    job_count = 0
-    started_job_count = 0
-    for build in active_builds:
-        if build['stage'] != 'finished':
-            jobs = get_jobs(build['id'])
-            for job in jobs:
-                if job['state'] != 'finished':
-                    job_count += 1
-                    if job['state'] != 'queued':
-                        started_job_count += 1
+    repos = get_repos(org)
+    org_build_count = 0
+    org_build_started_count = 0
+
+    for repo in repos:
+        repo_builds = get_active_builds(org, repo)
+        logger.debug("--->" + repo)
+
+        repo_build_total, num_started = repo_active_build_count(repo_builds)
```

Comment:
Not here because the list of builds is captured by `get_active_builds` above. That filters out anything that's `finished`.